enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -47,6 +48,10 @@ import (
 	fcmodels "github.com/firecracker-microvm/firecracker-go-sdk/client/models"
 )
 
+var (
+	forceDebugMode = flag.Bool("debug_firecracker_force_debug_mode", false, "Whether to force firecracker debug_mode, streaming firecracker logs to the console. Not intended for production use.")
+)
+
```

Comment:
do we need to introduce this here? Seems like debugmode can already be set via the opts passed into this class, wondering why we don't set this via a config flag or a flag in vmstart (if that's where you need it)?

Reply:
I found this useful while debugging both vmstart and the unit tests, which would have been a lot of places to set `DebugMode: true`. But this may not be super useful going forward -- removing for now